source/_docs/configuration/yaml.markdown

```diff
@@ -3,21 +3,29 @@ title: "YAML"
 description: "Details about YAML to configure Home Assistant."
 ---
 
-Home Assistant uses the [YAML](https://yaml.org/) syntax for configuration. YAML might take a while to get used to but is really powerful in allowing you to express complex configurations.
+Home Assistant can be configured in two ways:
 
-While more and more integrations are configured through the UI, for some, you will add code in your `configuration.yaml` file to specify its settings.
+1. Through `configuration.yaml` and other [YAML](https://yaml.org/) files.
+2. Through the web user interface (UI), which stores the configuration in JSON files in the `.storage` folder.
 
-The following example entry assumes that you would like to set up the [notify integration](/integrations/notify) with the [pushbullet platform](/integrations/pushbullet).
+Traditionally, all configuration was managed in YAML files, but as explained in the 2020 blog post ["The future of YAML"](https://www.home-assistant.io/blog/2020/04/14/the-future-of-yaml/), there's been a shift towards allowing more and more configuration to be done in the UI. For many integrations, you can still choose to use YAML if you like, but some things can only be configured in the web UI and some things only in YAML. Refer to the documentation for the specific integration for details.
```

Comment:
```suggestion
Traditionally, all configuration was managed in YAML files, but as explained in the 2020 blog post ["The future of YAML"](/blog/2020/04/14/the-future-of-yaml/), there's been a shift towards allowing more and more configuration to be done in the UI. For many integrations, you can still choose to use YAML if you like, but some things can only be configured in the web UI and some things only in YAML. Refer to the documentation for the specific integration for details.
```

- Relative links.
- "For many integrations, you can still choose to use YAML if you like" -> This is not true, we don't allow for those. More information can be found in ADR-0010 and in the blog post, you have linked.